### DISCUSSION TITLE
Not an Issue Just a Notation

### DISCUSSION THREAD

**Discussion by guarddog13** [gtxaspec/wz_mini_hacks#262] [General]
If you use the rasp pi imager you can format any size SD card to FAT32 with no issue, FYI.

**Comment by baudneo** [comment]
Sick, thanks for the idea.

**Comment by guarddog13** [comment]
Lol I use 128GB cards with this hack on all my cameras. I have 7 cameras running this on agentDVR with a deepstack server. I keep my recording for the SD cards and use the NVR just for people detection.

**Comment by gtxaspec** [comment]
@guarddog13 you can also check and create discussions here:

https://github.com/gtxaspec/wz_mini_hacks/discussions

just in case there is something that is not an issue

**Comment by gtxaspec** [comment]
converting to discussion for visibility
